content/docs/rfcs/14/README.md

```diff
@@ -1,120 +1,102 @@
 ---
 slug: 14
 title: 14/WAKU2-MESSAGE
-name: Waku v2 Message
-status: draft
+name: Waku v2 message RFC
+status: stable
+category: Standards Track
+tags: waku/core-protocol
 editor: Oskar Thorén <oskar@status.im>
 contributors:
   - Sanaz Taheri <sanaz@status.im>
   - Aaryamann Challani <aaryamann@status.im>
+  - Lorenzo Delgado <lorenzo@status.im>
 ---
 
-This specification provides a way to encapsulate messages sent over Waku with specific information security goals.
+# Abstract
+
+Waku v2 is a family of modular peer-to-peer protocols for secure communication. These protocols are designed to be secure, privacy-preserving, censorship-resistant, and able to run in resource-restricted environments. At a high level, Waku v2 implements a Pub/Sub messaging pattern over libp2p and adds capabilities to it.
+
+The present document specifies the Waku v2 message format and a way to encapsulate the messages sent with specific information security goals.
 
 # Motivation
 
-When sending messages over Waku there are multiple concerns:
+When sending messages over Waku, there are multiple concerns:
+
 - We may have a separate encryption layer as part of our application
-- We may want to provide efficient routing for resource restricted devices
+- We may want to provide efficient routing for resource-restricted devices
 - We may want to provide compatibility with Waku v1 envelopes
 - We may want payloads to be encrypted by default
 - We may want to provide unlinkability for metadata protection
 
 This specification attempts to provide for these various requirements.
 
-# WakuMessage
+# Semantics
+
+## Waku Message
 
-A `WakuMessage` is what is being passed around by the other protocols, such as WakuRelay, WakuStore, and WakuFilter.
+A waku message is constituted by the combination of data and attributes that, for example, a *publisher* sends to a *topic* and is eventually delivered to *subscribers*.
```

Comment:
How `attribute` is distinguished from `data`?

Reply:
You have used these two terms, `data` and `attribute`, then the following subsection is titled `message attributes.`, so it comes across that the `Message data` section is missing. That is why I asked how you distinguish these two. Hope that clarifies it.